packages/eslint-config-react-native-community/package.json

```diff
@@ -10,18 +10,18 @@
   },
   "dependencies": {
     "@react-native-community/eslint-plugin": "^1.0.0",
-    "@typescript-eslint/eslint-plugin": "^1.5.0",
-    "@typescript-eslint/parser": "^1.5.0",
-    "babel-eslint": "10.0.3",
-    "eslint-config-prettier": "^6.0.0",
+    "@typescript-eslint/eslint-plugin": "^2.25.0",
+    "@typescript-eslint/parser": "^2.25.0",
+    "babel-eslint": "10.1.0",
+    "eslint-config-prettier": "^6.10.1",
     "eslint-plugin-eslint-comments": "^3.1.2",
     "eslint-plugin-flowtype": "2.50.3",
     "eslint-plugin-jest": "22.4.1",
-    "eslint-plugin-prettier": "2.6.2",
-    "eslint-plugin-react": "7.16.0",
-    "eslint-plugin-react-hooks": "^2.0.1",
+    "eslint-plugin-prettier": "3.1.2",
+    "eslint-plugin-react": "7.19.0",
+    "eslint-plugin-react-hooks": "^2.5.1",
     "eslint-plugin-react-native": "3.8.1",
-    "prettier": "1.17.0"
+    "prettier": "^2.0.2"
   },
   "peerDependencies": {
     "eslint": ">=5"
```

Comment:
Does this need to be updated as well, or is ESLint 5 still supported?

Reply:
from my tests, it seems eslint >=6 is required. Thanks for that